### ISSUE TITLE
Import removes owl:equivalentProperty annotation

### ISSUE THREAD

**Issue by pols12** [protegeproject/webprotege#497]
1. Create a property with `myTitle` as `rdfs:label`.
1. Define `myTitle` as `owl:equivalentProperty` of `http://purl.org/dc/terms/title` (this will create `owl:equivalentProperty` as Annotation Property).
1. Export the projet. (You can read the file: annotation is well imported.)
1. Import the file as a new project.

Actual result: `owl:equivalentProperty` is well created as an Annotation Property, but myTitle has no `owl:equivalentProperty` annotation defined.

**Comment by matthewhorridge** [comment]
@pols12 please can you confirm that you're working in WebProtege (and not the desktop version of Protege).

**Comment by pols12** [comment]
Indeed, I work with [online-provided version of WebProtégé](https://webprotege.stanford.edu).

**Comment by matthewhorridge** [comment]
Thanks.  Please can you list the exact steps that you perform in step 2 above?

**Comment by pols12** [comment]
Sorry for being unclear. Details of step 2:
1. In Property Hierarchy panel, I click on myTitle property in order to edit it.
1. In edit panel, under Annotations header, I click on “Enter property” placeholder in order to add an annotation.
1. I type “owl:equivalentProperty” (an information box says “<quote>**owl:equivalentProperty** is a **new** property name. To continue, enter a value for the property (class name, number etc.) and press the TAB key</quote>”).
1. I press Tab key to go to “Enter value” field.
1. I type “http://purl.org/dc/terms/title”
1. I press Tab key again to validate the annotation addition.
